app/src/main/kotlin/codes/chrishorner/socketweather/data/ApiModels.kt

```diff
@@ -43,13 +45,18 @@ data class DateForecast(
     val short_text: String?,
     val icon_descriptor: String,
     val rain: Rain,
+    val uv: UV,
     val now: CurrentInformation? = null
 )
 
 data class Rain(val amount: Amount, val chance: Int) {
   data class Amount(val min: Float?, val max: Float?, val units: String)
 }
 
+data class Wind(val speed_knot: Int, val speed_kilometre: Int, val direction: String)
```

Comment:
Doesn't look like we're making use of `speed_knot` anywhere at the moment. Let's remove it for now
```suggestion
data class Wind(val speed_kilometre: Int, val direction: String)
```